html/pfappserver/root/src/views/Configuration/mfas/_components/FormTypeTOTP.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <base-form
+    :form="form"
+    :meta="meta"
+    :schema="schema"
+    :isLoading="isLoading"
+  >
+    <form-group-identifier namespace="id"
+      :column-label="$i18n.t('Name')"
+      :disabled="!isNew && !isClone"
+    />
+
+    <form-group-radius-mfa-method namespace="radius_mfa_method"
+      :column-label="$i18n.t('RADIUS OTP Method')"
+      :text="$i18n.t('Define the method to be used in RADIUS to trigger OTP.')"
+    />
+
+    <form-group-split-char namespace="split_char"
+      :column-label="$i18n.t('Character separator')"
+      :text="$i18n.t('Please specify the char to split password field to get the code.')"
```

Comment:
```suggestion
      :text="$i18n.t('The char with which the password and the code are split during RADIUS authentication.')"
```

Make sure my RADIUS context applies too before applying it but it does from what I can tell